tests/test_find_executable.py

```diff
@@ -0,0 +1,5 @@
+from ..src.PartielsPy.Partiels import Partiels
+
+
+def test_find_executable():
+    assert Partiels().getExecutablePath() is not None, "Partiels Executable not found"
```

Comment:
It might be worth testing using `PATH` & `PARTIELS_PATH` variables and also to test the failure